DependencyInjection/SuluFormExtension.php

```diff
@@ -223,6 +225,11 @@ public function load(array $configs, ContainerBuilder $container): void
         }
 
         if (\class_exists(\EWZ\Bundle\RecaptchaBundle\Form\Type\EWZRecaptchaType::class)) {
+            if (3 == $container->getParameter('ewz_recaptcha.version')) {
```

Comment:
@bodabodah I've testet your suggestion with the latest version of sulu/form-bundle 2.5.2, excelwebzone/recaptcha-bundle 1.5.39 and symfony 6.3.4 but ran into the following issues.

`You have requested a non-existent parameter "ewz_recaptcha.version".`

Reply:
My suggestion works fine with : 
sulu/form-bundle 2.4.5, excelwebzone/recaptcha-bundle 1.4.32 and symfony 5.4.12

Are you sure that the EWZRecaptchaBundle is enabled in your installation ?

Reply:
@tniebergall I just figured it out. The parameter is defined in EWZRecaptchaBundle. So we get this error when EWZRecaptchaBundle is registerd after SuluFormBundle. I made a fix. can you test again ?

The fix just removes the error message but to use recaptcha v3 we have to register EWZRecaptchaBundle before SuluFormBundle.